<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoUpdate</class>
 <widget class="QWidget" name="AutoUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxAutoUpdate">
     <property name="title">
      <string>AutoUpdate</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="5" column="0">
       <widget class="QPushButton" name="btnCheck">
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="gLayTarget" columnstretch="1,2">
        <item row="0" column="0">
         <widget class="QLabel" name="labelTargetVersion">
          <property name="text">
           <string>Target Version</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditTargetVersion"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gLayCurrent" columnstretch="1,2">
        <item row="0" column="0">
         <widget class="QLabel" name="labelCurrentVersion">
          <property name="text">
           <string>Current Version</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditCurrentVersion"/>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBoxUpdateCheck">
        <property name="text">
         <string>Update Check</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelURL">
          <property name="text">
           <string>URL</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditURL"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
